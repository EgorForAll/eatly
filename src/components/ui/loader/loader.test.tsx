import {screen, render} from "@testing-library/react";
import Loader from "./loader";
import '@testing-library/jest-dom';


test('Loader renders correctly', () => {
    render(<Loader />)
    const loader = screen.getByTestId('loader');
    expect(loader).toMatchSnapshot();
})