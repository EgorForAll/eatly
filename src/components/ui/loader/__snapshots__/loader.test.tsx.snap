// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<span
  class="loader"
  data-testid="loader"
/>
`;
